media-libs/smpeg/files/smpeg-0.4.4-format.patch

```diff
@@ -1,5 +1,5 @@
---- plaympeg.c.old	2014-09-04 08:55:10.646132150 +0200
-+++ plaympeg.c	2014-09-04 08:56:57.465210820 +0200
+--- a/plaympeg.c	2014-09-04 08:55:10.646132150 +0200
++++ b/plaympeg.c	2014-09-04 08:56:57.465210820 +0200
```

Comment:
That `sed` function is really helpful, thanks :)
I've updated all patches which means I've also updated a few which i haven't touched yet. I guess since I'm already fixing them, I'm not going to leave the others out just because they were working.